src/Umbraco.Web.UI.Client/src/views/datatypes/create.html

```diff
@@ -24,13 +24,19 @@ <h5><localize key="create_createUnder">Create an item under</localize> {{current
             </ul>
         </div>
 
-        <div class="umb-pane" ng-if="model.creatingFolder">
+        <div class="umb-pane" ng-show="model.creatingFolder">
             <form novalidate name="createFolderForm"
                 ng-submit="createContainer()"
                 val-form-manager>
 
                 <umb-control-group label="Enter a folder name" hide-label="false">
-                    <input type="text" name="folderName" ng-model="model.folderName" class="umb-textstring textstring input-block-level" required umb-auto-focus />
+                    <input 
+                        type="text" 
+                        name="folderName" 
+                        ng-model="model.folderName" 
+                        class="umb-textstring textstring input-block-level"
+                        focus-when="{{model.creatingFolder}}" 
+                        required umb-auto-focus />
```

Comment:
@madsrasmussen do this need `umb-auto-focus` directive now it has `focus-when`. It is not the case for media types.

Reply:
Nice catch! 💯 

It is removed 👍